presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -645,8 +648,10 @@ public void success(TaskInfo value)
                 try {
                     long currentRequestStartNanos;
                     synchronized (HttpRemoteTask.this) {
+                        //noinspection FieldAccessNotGuarded
```

Comment:
Because IntelliJ doesn't understand that "this" in the annotation is the same as "HttpRemoteTask.this"